### DISCUSSION TITLE
Handling of too-big cache

### DISCUSSION THREAD

**Discussion by HebaruSan** [actions/cache#486] [Q&A]
I realized belated this should have been a Discussion, about to delete my comment:

> What does this action do if the contents of the individual cache folder is 5 GiB or greater, not just the total collection of all saved caches? Does it not save it at all?

Thanks to @PathogenDavid for the answer!

https://github.com/actions/cache/issues/342#issuecomment-743814709

> @HebaruSan It won't save anything at all.
> 
> https://github.com/actions/toolkit/blob/73d5917a6b5ea646ac3173cfceb727ee914ff6ed/packages/cache/src/cache.ts#L166-L175

**Comment by PathogenDavid** [comment]
No problem!
